common/include/pcl/pcl_macros.h

```diff
@@ -301,18 +301,6 @@ pcl_round (float number)
 #define FIXED(s) \
   std::fixed << s << std::resetiosflags(std::ios_base::fixed)
 
-#ifndef ERASE_STRUCT
```

Comment:
Please deprecate before removing: https://stackoverflow.com/a/57479189

TLDR: Create a variable in `namespace pcl::detail`, deprecate it. Refer to that variable (and ignore it using a comma operator) in the macro

Reply:
Restored, will reassess deprecating these later